cms/standard_pages/models.py

```diff
@@ -38,3 +40,76 @@ class InformationPage(BasePage):  # type: ignore[django-manager-missing]
         index.SearchField("summary"),
         index.SearchField("content"),
     ]
+
+
+class IndexPage(BasePage):  # type: ignore[django-manager-missing]
+    template = "templates/pages/index_page.html"
+
+    parent_page_types: ClassVar[list[str]] = ["home.HomePage", "IndexPage", "InformationPage"]
```

Comment:
```suggestion
    parent_page_types: ClassVar[list[str]] = ["home.HomePage", "IndexPage"]
```

I don't think we should allow generic index pages under information pages.

Reply:
Done in [Add content to search indexing + stop allowing Index pages under Info pages](https://github.com/ONSdigital/dis-wagtail/pull/78/commits/5b538cd7b44fbd42b493e18b9581fff40d28b84b)